### PR TITLE
Fix test class names

### PR DESCRIPTION
**Pull request by chanseokoh**
Tests don't run if not ending with `Test`.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProject.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
  * terminate.
  */
 @RunWith(SWTBotJunit4ClassRunner.class)
-public class DebugNativeAppEngineStandardProject extends AbstractProjectTests {
+public class DebugNativeAppEngineStandardProject extends AbstractProjectTest {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProject.java]: DebugNativeAppEngineStandardProjectTest

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
 /**
  * Common infrastructure for workbench-based tests that create a single project.
  */
-public class AbstractProjectTests {
+public class AbstractProjectTest {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java]: I'm surprised the name of the superclass matters, but c'et la vie. However this class should be abstract (or renamed).

**Comment by akerekes** [comment]
I haven't tested it, but JUnit 4 doesn't care about the name of the class,
it only looks for the annotations to be present. The reason these tests
aren't executed still is probably the Maven Surefire plugin's default
inclusion filter:
http://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java (hunk 2 above)
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java]: I did another test to verify this: as long as the inheriting class ends with `Test`, the name of the superclass doesn't matter.

Anyway, I renamed this to `BaseProjectTest`. (Not `Abstract`.)

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProject.java (hunk 1 above)
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProject.java]: Good catch.

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java [hunk 3]
```diff
@@ -30,7 +30,7 @@
 /**
  * Common infrastructure for workbench-based tests that create a single project.
  */
-public class AbstractProjectTest {
+public class BaseProjectTest {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java]: could still make this abstract if you like

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java]: There's no abstract method in this class, so I'll keep it `BaseProjectTest`. (It's not my code, so I don't immediately see how I could convert this to an abstract class.)

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java]: abstract classes are not required to have abstract mthods

**Pull request merged**